webapp/boards/tsconfig.json

```diff
@@ -18,25 +18,17 @@
         "sourceMap": true,
         "allowJs": true,
         "resolveJsonModule": true,
-        "incremental": false,
         "baseUrl": ".",
         "outDir": "dist",
         "noEmit": true,
-        "typeRoots": [ "./src/types", "./node_modules/@types"],
-        "paths": {
-            "@mattermost/types/*": ["../platform/packages/types/src/*"],
-            "@mattermost/client": ["../platform/client/src"],
-            "@mattermost/components": ["../platform/components/src"],
-            // "mattermost-redux/*": ["../channels/src/packages/mattermost-redux/src/*"],
-            // "reselect": ["../channels/src/packages/reselect/src"]
-        }
+        "composite": true
     },
     "include": [
-      "**/*"
+        "./i18n/*.json",
+        "./src/**/*",
     ],
-    "exclude": [
-        ".git",
-        "!node_modules/@types",
-        "dist",
+    "references": [
```

Comment:
Project references are used to let TS projects find each other more easily. Now that the missing `types` and `typesVersions` fields are added to the platform packages' package.jsons, we can use these without having to manually specify the file paths